bin/server/cmd/root.go

```diff
@@ -119,6 +123,7 @@ func Execute() error {
 func init() {
 	rootCmd.PersistentFlags().StringVar(&host, "host", "http://localhost", "host where Bytebase backend is accessed from, must start with http:// or https://. This is used by Bytebase to create the webhook callback endpoint for VCS integration")
 	rootCmd.PersistentFlags().IntVar(&port, "port", 80, "port where Bytebase backend is accessed from. This is also used by Bytebase to create the webhook callback endpoint for VCS integration")
+	rootCmd.PersistentFlags().IntVar(&datastorePort, "datastore-port", 0, "port of datastore instance for storing Bytebase data")
```

Comment:
I consider to remove it completely because it's an implementation detail. Ideally, it's transparent to the user whether we are using SQLite or PG, so from their perspective, the interface should be exactly the same.

Reply:
Done